roles/devscripts/tasks/111_verify_cluster.yml

```diff
@@ -19,7 +19,7 @@
   ansible.builtin.import_tasks: set_cluster_fact.yml
 
 - name: Login to the deployed OpenShift cluster.
-  kubernetes.core.k8s_auth:
+  community.okd.openshift_auth:
```

Comment:
with ansible-lint, it fails, because it apparently wants that one instead (probably because kubernetes.core.k8s_auth is mostly a wrapper around that one?)

Reply:
(note that the pod seems to apply different rules, or maybe uses another version - since we're not seeing this in the prow job o_O)

Reply:
pushed a one-line patch to see if it triggers the ansible-lint issue. I really don't remember if it was local on my env, or indeed somewhere in zuul... "too old" for my brain.